bin/git-rename-file

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
```

Comment:
Let's add https://github.com/tj/git-extras/blob/e1a4e7195d54cf3e0a931bb53a2584697ba431f0/bin/git-rename-remote#L3 for new command

Reply:
I'm good with this.